front/migrations/20241101_workspace_keys_incident.ts

```diff
@@ -0,0 +1,211 @@
+import { parse } from "csv-parse";
+import * as fs from "fs";
+import { exit } from "process";
+
+import { sendEmail } from "@app/lib/api/email";
+import { getMembers } from "@app/lib/api/workspace";
+import { Authenticator } from "@app/lib/auth";
+import { Workspace } from "@app/lib/models/workspace";
+import { getStripeClient } from "@app/lib/plans/stripe";
+import type { Logger } from "@app/logger/logger";
+import { makeScript } from "@app/scripts/helpers";
+
+const MIN_AMOUNT_FOR_REIMBURSEMENT = 1;
+const CURRENCY = "eur";
+
+const sendIncidentEmailToAdmins = async (
+  workspaceId: number,
+  cost_in_cents: number,
+  reimbursement_in_cents: number,
+  logger: Logger
+) => {
+  const message = {
+    from: {
+      name: "Dust team",
+      email: "team@dust.tt",
+    },
+    subject: `[Dust] Workspace keys incident`,
```

Comment:
Provided alternative subject and body on slack

Reply:
Saw it, updated.